test/integration/examples/apiserver_test.go

```diff
@@ -90,9 +92,14 @@ func TestAggregatedAPIServer(t *testing.T) {
 			"--kubeconfig", wardleToKASKubeConfigFile,
 		})
 		if err := wardleCmd.Execute(); err != nil {
-			t.Fatal(err)
+			errs <- err
```

Comment:
I think you need to `close(errs)`  if no error occurs, otherwise `err = <-errs` would block indefinitely.

Reply:
Thanks. I'll remove the channel and change it to err per @liggitt's suggestion.